index.bs

```diff
@@ -1368,6 +1419,12 @@ Elected Groups Vacated Seats</h5>
 			and the maximum number corresponds to all unoccupied seats.
 			Except for the number of available seats and the length of the terms,
 			the <a href="#AB-TAG-elections">usual rules for Advisory Board and Technical Architecture Group Elections</a> apply.
+
+		<li>
+			If seats are vacated due to a successful [=vote of no confidence=],
+			the [=Team=] <em class=rfc2119>must</em> organize an election,
+			under the same condition as for individually vacated seats,
+			unless the next regularly scheduled election is fewer than three months away.
```

Comment:
I'm not clear on the three month choice.  That means that you might win an election for a 3 month and a day term.  That's pretty pointless, I'd go with six.

```suggestion
			unless the next regularly scheduled election for the vacancy is less than six months away,
			in which case the remainder of the term is extended by two years.
```

Reply:
Works for me. I'll let chairs decide if there's consensus on this point or not.

Reply:
Three months is not the term being elected for, it's the time remaining until the next election: the term might extend for another year on top of those three months.

Three months is the existing threshold we have for not scheduling a special election because it takes approximately that long to organize the regular election anyway.

So I think we keep this threshold at three months, and if we want something to handle excessively short terms, then that would be a separate point.

Reply:
There are two things: when the election is held (at vacancy vs. on the regular schedule) and what the term is (the remainder of the vacated term, or something else).  Those need to both be specified clearly.

I would prefer that vacancies not exist for too long, but maybe that is better addressed in #1032, rather than here.

As for the term, the assumption will be that the term is for the period of vacancy, unless otherwise specified.  Personally, I don't like the idea that you might get term+N unless N is sufficiently small (3 months is maybe small enough).

Reply:
> As for the term, the assumption will be that the term is for the period of vacancy, unless otherwise specified.  Personally, I don't like the idea that you might get term+N unless N is sufficiently small (3 months is maybe small enough).

Agreed. Filed as https://github.com/w3c/process/issues/1033